apps/agreement/contracts/Agreement.sol

```diff
@@ -0,0 +1,1350 @@
+/*
+ * SPDX-License-Identitifer:    GPL-3.0-or-later
+ */
+
+pragma solidity 0.4.24;
+
+import "@aragon/os/contracts/apps/AragonApp.sol";
+import "@aragon/os/contracts/apps/disputable/IAgreement.sol";
+import "@aragon/os/contracts/apps/disputable/IDisputable.sol";
+import "@aragon/os/contracts/common/ConversionHelpers.sol";
+import "@aragon/os/contracts/common/SafeERC20.sol";
+import "@aragon/os/contracts/common/TimeHelpers.sol";
+import "@aragon/os/contracts/lib/token/ERC20.sol";
+import "@aragon/os/contracts/lib/math/SafeMath.sol";
+import "@aragon/os/contracts/lib/math/SafeMath64.sol";
+
+import "./lib/BytesHelper.sol";
+import "./staking/Staking.sol";
+import "./staking/StakingFactory.sol";
+
+
+contract Agreement is IAgreement, AragonApp {
+    using SafeMath for uint256;
+    using SafeMath64 for uint64;
+    using SafeERC20 for ERC20;
+    using BytesHelper for bytes;
+
+    /* Arbitrator outcomes constants */
```

Comment:
Just wondering, we don't have an enum type defined in `IArbitrator` for this, right?

Reply:
We don't :/